nss-day-cohort-41/nutshell-my-pretty-rose-tree
# update to readme and added ERD image

# Description
Updated readme and added ERD image
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
fetch branch `ak-first-branch` and checkout to branch. pull up readme and make sure Nutshell ERD image file is in images.
# Checklist:
